packages/cli/src/templates/expo.ts

```diff
@@ -0,0 +1,237 @@
+            config.extends = "expo/tsconfig.base";
+            // @ts-expect-error structure of config file subject to change
+            config.compilerOptions = {
+              moduleResolution: "bundler",
+              strict: true,
+            };
+            return config;
+          });
+
+          const packageJsonFilePath = `${cwd}/package.json`;
+          await modifyJsonFile(packageJsonFilePath, (config) => {
+            // @ts-expect-error structure of config file subject to change
+            config.scripts.format = "prettier --write .";
+            return config;
+          });
+
+          try {
+            const babelConfigFilePath = `${cwd}/babel.config.js`;
+            const data = await readFile(babelConfigFilePath, "utf8");
+            const updatedConfig = data.replace(
+              `presets: ['babel-preset-expo'],`,
+              `presets: ['babel-preset-expo'],
+              plugins: [
+                ["@babel/plugin-transform-private-methods", { "loose": true }],
+              ],`,
+            );
+            await writeFile(babelConfigFilePath, updatedConfig, "utf8");
+          } catch (error) {
+            console.error(
+              "Error reading or writing the babel.config.js file:",
+              error,
+            );
+          }
+
+          await writeFile(`${cwd}/App.tsx`, APP_TSX_CONTENT, "utf8");
+          await writeFile(
+            `${cwd}/metro.config.js`,
+            METRO_CONFIG_CONTENT,
+            "utf8",
+          );
+          await writeFile(
+            `${cwd}/PatientComponent.tsx`,
+            PATIENT_COMPONENT_CONTENT,
+            "utf8",
+          );
+
+          await writeFile(
+            `${cwd}/.prettierignore`,
+            PRETTIER_IGNORE_CONTENT,
+            "utf8",
+          );
+        },
+      },
+      {
+        title: "Add dependencies",
+        task: async ({ options: { cwd, packageManager } }) => {
+          await packageManager.install(cwd);
+          await packageManager.add(
+            cwd,
+            "@bonfhir/core",
+            "@bonfhir/query",
+            "@bonfhir/react",
+            "@bonfhir/gluestack-ui",
+            "@gluestack-style/react",
+            "@gluestack-ui/themed",
+            "@gluestack-ui/config",
+            "@legendapp/motion",
+            "react-dom",
+            "react-native-web",
+            "@expo/webpack-config",
+          );
+
+          await packageManager.addDev(cwd, "@types/react-native", "prettier");
+        },
+      },
+      {
+        title: "Format all files",
+        task: async ({ options: { cwd, packageManager } }) => {
+          await packageManager.runPrettier(cwd, ".");
+        },
+      },
+      {
+        title: "Commit bonFHIR setup changes",
+        task: async ({ options: { cwd } }) => {
+          await execAsync(`git add .`, { cwd });
+          await execAsync(`git commit -m "Setup bonFHIR"`, { cwd });
```

Comment:
`- a` only commits already tracked files, since we add new files, I slip the command in two